src/main/java/edu/wisc/my/restproxy/dao/GenericRestLookupDao.java

```diff
@@ -4,6 +4,10 @@
 
 import org.springframework.http.HttpMethod;
 
+/**
+ * @deprecated see {@link RestProxyDao} instead
+ */
+@Deprecated
```

Comment:
Yeah, probably a major version is in order since it would be backwards-incompatible.